qdrant-landing/content/blog/datatalk-club-podcast-plug.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Navigating challenges and innovations in search technologies"
+draft: false
+slug: navigating-challenges-innovations
+short_description: Podcast on search and LLM with Datatalk.club
+description: Podcast on search and LLM with Datatalk.club
+date: 12.01.2024
```

Comment:
the date should be specified in the format like this: `2022-09-26T15:39:53.751Z`